### PR TITLE
Append session ID to avoid manual login

### PR DESCRIPTION
**Pull request by KochTobi**

### CODE REVIEW

File: src/main/java/life/qbic/omero/BasicOMEROClient.java [hunk 1]
```diff
@@ -247,7 +247,7 @@ public String getImageDownloadLink(long imageID) {
       ImageData image = browse.getImage(this.ctx, imageID);
       disconnect();
       if (image.getFormat() != null) {
-        res = hostname + "/omero/webgateway/archived_files/download/" + imageID + "/";
+        res = hostname + "/omero/webgateway/archived_files/download/" + imageID + "?server=1&bsession=" + this.sessionId;
```
Comment (sven1103) [inline src/main/java/life/qbic/omero/BasicOMEROClient.java:250]: NIT: a more declarative name is be favored

**Review by sven1103 — DISMISSED** [review]
LGTM

File: src/main/java/life/qbic/omero/BasicOMEROClient.java (hunk 1 above)
Comment (KochTobi) [reply src/main/java/life/qbic/omero/BasicOMEROClient.java]: instead of result you mean? Will change that, good point.

Comment (KochTobi) [reply src/main/java/life/qbic/omero/BasicOMEROClient.java]: b50b3e3

**Pull request merged**